src/Core/Settings/GlobalSettings.cs

```diff
@@ -146,6 +147,12 @@ public BaseServiceUriSettings(GlobalSettings globalSettings)
         public string Vault { get; set; }
         public string VaultWithHash => $"{Vault}/#";
 
+        public string CloudVault
+        {
+            get => string.IsNullOrWhiteSpace(_cloudVault) ? _globalSettings.SelfHosted ? "https://vault.bitwarden.com" : Vault : _cloudVault;
```

Comment:
♻️ Want to make sure this cleanup is filed as a task somewhere.

Reply:
What kind of cleanup? Not relying on the `Vault` variable for a default value if `CloudWebVault` is not present?